.travis/fedora/travis-tasks.sh

```diff
@@ -75,7 +75,7 @@ $RETRY_CMD dnf -y install --nogpgcheck \
 
 # Also install the python2-libmodulemd if it was built for this release
 # the ||: at the end instructs bash to consider this a pass either way.
-dnf -y install --nogpgcheck \
+$RETRY_CMD dnf -y install --nogpgcheck \
```

Comment:
That was actually intentional, since at this point we know the python2 library won't usually exist, so there's no reason to try three times to install it.